### PR TITLE
New package for debugging settings and removing Spec debugger filtering settings code

### LINKED ISSUE #3692: Debugger settings are mixed between model, settings and ui concerns
This constraints further evolution/refatoring of debugging tools and models;

### PR DESCRIPTION
**Pull request by StevenCostiou**
- Moving debugging settings classes from debugging model package to a dedicated debugging settings package (its not about the debugging model!)

- Temporary removing old-spec-related debugging filters settings which is blocking for (r)evolution of the spec debugger

Fixes#3692

### CODE REVIEW

File: src/Debugging-Settings/DebugSystemSettings.class.st [hunk 1]
```diff
@@ -4,7 +4,7 @@ Settings for debugging
 Class {
 	#name : #DebugSystemSettings,
 	#superclass : #Object,
-	#category : #'Debugger-Model-Settings'
+	#category : #'Debugging-Settings'
```
Comment (jecisc) [inline src/Debugging-Settings/DebugSystemSettings.class.st:7]: Package renaming should include removal of the previous package and update of the baseline.

**Review by jecisc — CHANGES_REQUESTED** [review]

File: src/Debugging-Settings/DebugSystemSettings.class.st (hunk 1 above)
Comment (StevenCostiou) [reply src/Debugging-Settings/DebugSystemSettings.class.st:7]: This is not renaming, this is a new package.
Baseline updated.

**Comment by Ducasse** [comment]
Green bar!

**Comment by Ducasse** [comment]
Green bar!

**Pull request merged**